### ISSUE TITLE
Crash fatal error

### ISSUE THREAD

**Issue by 2020karaulov2020** [theAsmodai/metamod-r#51]
https://dev-cs.ru/threads/700/post-100131

**Comment by theshadowz** [comment]
https://github.com/dreamstalker/rehlds/blob/462484a644c0de43bd74fd030ca5c15fde00892c/rehlds/engine/pr_edict.cpp#L402-L405

**Comment by 2020karaulov2020** [comment]
Yes this message from rehlds, but crash caused by metamod-r!

**Comment by 2020karaulov2020** [comment]
> https://github.com/dreamstalker/rehlds/blob/462484a644c0de43bd74fd030ca5c15fde00892c/rehlds/engine/pr_edict.cpp#L402-L405

Yes this message from rehlds, but this crash caused by metamod-r!

**Comment by theshadowz** [comment]
> https://dev-cs.ru/threads/700/post-100131

Attentively read the logs you attached on this link, then make search by error-log on rehlds' repo and you will see a condition which execution sends the error
